Most of xfs recompiles during "make bzImage" because
linux/include/linux/version.h is touched.

The patch below eliminates most of the extra recompile
and changes the boot message from:

SGI XFS for Linux 2.5.73-mm1 with no debug enabled
to
SGI XFS for Linux with no debug enabled

diff -u linux-2.5.73-mm1/fs/xfs/linux/xfs_version.h  linux/fs/xfs/linux/xfs_version.h
--- linux-2.5.73-mm1/fs/xfs/linux/xfs_version.h 2002-12-12 04:36:32.000000000 -0500
+++ linux/fs/xfs/linux/xfs_version.h    2003-06-29 12:16:48.000000000 -0400
@@ -39,8 +39,6 @@
 #ifndef __XFS_VERSION_H__
 #define __XFS_VERSION_H__

-#include <linux/version.h>
-
-#define XFS_VERSION_STRING "for Linux " UTS_RELEASE
+#define XFS_VERSION_STRING "for Linux"

 #endif /* __XFS_VERSION_H__ */

-- 
Randy Hron
http://home.earthlink.net/~rwhron/kernel/bigbox.html


